### PR TITLE
fredapi recipe

### PR DESCRIPTION
**Pull request by tk3369**

### CODE REVIEW

File: recipes/fredapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 11d8143a7fa8a3a7c6b0ef351a4dd5da0c287a152eb778a6b1481a39ffdcbf01
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
```
Comment (isuruf) [inline recipes/fredapi/meta.yaml]: Can this be `noarch: python`?

File: recipes/fredapi/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+Apache License
```
Comment (isuruf) [inline recipes/fredapi/LICENSE]: Can you open an issue or make a PR upstream to add the license file to the source distribution?

File: recipes/fredapi/meta.yaml (hunk 1 above)
Comment (tk3369) [reply recipes/fredapi/meta.yaml]: I guess so.  Will update and try again.

File: recipes/fredapi/LICENSE (hunk 2 above)
Comment (tk3369) [reply recipes/fredapi/LICENSE]: This is a copy of the same license.

File: recipes/fredapi/meta.yaml [hunk 3]
```diff
@@ -10,8 +10,7 @@ source:
   sha256: 11d8143a7fa8a3a7c6b0ef351a4dd5da0c287a152eb778a6b1481a39ffdcbf01
 
 build:
-  number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (isuruf) [inline recipes/fredapi/meta.yaml]: Why did you remove this line?

File: recipes/fredapi/LICENSE (hunk 2 above)
Comment (isuruf) [reply recipes/fredapi/LICENSE]: I meant the sdist uploaded to PyPI which doesn't include LICENSE file because it is not listed in the MANIFEST.in file

Comment (tk3369) [reply recipes/fredapi/LICENSE]: Submitted upstream PR https://github.com/mortada/fredapi/pull/22

Comment (tk3369) [reply recipes/fredapi/LICENSE]: fredapi's sdist (0.4.2) has now been updated with LICENSE file.  Are we ready to merge this PR now?

Comment (tk3369) [reply recipes/fredapi/LICENSE]: @isuruf ping... thanks for your help

File: recipes/fredapi/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{% set name = "fredapi" %}
+{% set version = "0.4.1" %}
```
Comment (isuruf) [inline recipes/fredapi/meta.yaml]: Do you want 0.4.1? If not, let's update this to 0.4.2

Comment (tk3369) [reply recipes/fredapi/meta.yaml]: @isuruf Good catch.  It should be 0.4.2.  Just committed.

**Comment by tk3369** [comment]
@isuruf I think it's all ready to merge.  Please review, thanks again!

File: recipes/fredapi/LICENSE (hunk 2 above)
Comment (isuruf) [reply recipes/fredapi/LICENSE]: Can you remove this file now? Conda-build will get it from the source directory.

**Comment by tk3369** [comment]
@isuruf the LICENSE file has bee removed from this PR.  The CI looks broken due to infrastructure issue.  Shall we merge or can you help re-run the build?   Your call.

circieci log as follows:
```
+ pushd /home/conda/staged-recipes/recipes
+ git fetch --force origin master:master
fatal: cannot run ssh: No such file or directory
fatal: unable to fork
Exited with code 1
```

**Comment by tk3369** [comment]
Hi @isuruf.  Just wondering if it's ready for merge.  Please let me know if any more changes are required.  Thanks again for your assistance.

**Comment by tk3369** [comment]
I don't understand why win_64 build failed.  The log is somewhat opaque...

```
##[command]"C:\windows\system32\cmd.exe" /D /E:ON /V:OFF /S /C "CALL "D:\a\_temp\64c8b21b-fd7d-4fd3-9b90-1ea1ee36f2d2.cmd""
Setting build arch. This is only useful when pretending to be on another arch, such as for rendering necessary dependencies on a non-native arch. I trust that you know what you're doing.
Setting build platform. This is only useful when pretending to be on another platform, such as for rendering necessary dependencies on a non-native platform. I trust that you know what you're doing.
No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
WARNING:conda_build.metadata:No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
INFO:conda_build.variants:Adding in variants from internal_defaults
Building fredapi with conda-forge/label/main
rendering D:\a\1\s\recipes\fredapi for win-win-64
Computed that there are 1 distributions to build from 1 recipes
Resolved dependencies, will be built in the following order:
    fredapi-0.4.2--target_win-64-on-win-win-64
Adding in variants from internal_defaults
Adding in variants from C:\Miniconda\conda_build_config.yaml
INFO:conda_build.variants:Adding in variants from C:\Miniconda\conda_build_config.yaml
INFO:conda_build.metadata:Attempting to finalize metadata for fredapi
Attempting to finalize metadata for fredapi
Traceback (most recent call last):
  File ".ci_support\build_all.py", line 130, in <module>
    build_all(args.recipes_dir, args.arch)
  File ".ci_support\build_all.py", line 68, in build_all
    build_folders(recipes_dir, new_comp_folders, arch, channel_urls)
  File ".ci_support\build_all.py", line 120, in build_folders
    conda_build.api.build(list(d.keys()), config=get_config(arch, channel_urls))
...
```

**Comment by mariusvniekerk** [comment]
Please rebase / merge current master

**Comment by tk3369** [comment]
Just rebased and force pushed:
```
$ git rebase upstream/master
First, rewinding head to replay your work on top of it...
Applying: sync
Applying: noarch build
Applying: added build number
Applying: put back build script
Applying: Update meta.yaml
Applying: Update meta.yaml
Applying: updated hash code
Applying: no need for LICENSE file anymore

$ git push -f
Username for 'https://github.com': tk3369
Password for 'https://tk3369@github.com': 
Counting objects: 7645, done.
Delta compression using up to 2 threads.
Compressing objects: 100% (4136/4136), done.
Writing objects: 100% (7645/7645), 1.99 MiB | 7.28 MiB/s, done.
Total 7645 (delta 3232), reused 7348 (delta 3001)
remote: Resolving deltas: 100% (3232/3232), completed with 31 local objects.
To https://github.com/tk3369/staged-recipes.git
 + d75d17de4...4a7c78263 fredapi -> fredapi (forced update)
```

**Comment by tk3369** [comment]
Given that it's a problem with Azure pipeline only, is it OK to just merge this PR?

File: .azure-pipelines/azure-pipelines-win.yml [hunk 5]
```diff
@@ -8,6 +8,7 @@ jobs:
       win:
         CONFIG: azure-win-64
         CF_MAX_PY_VER: 37
+        CONDA_BLD_PATH: D:\bld
```
Comment (chrisburr) [inline .azure-pipelines/azure-pipelines-win.yml]: @mariusvniekerk, can these changes be merged into master? Or should they be removed before merging?

Comment (tk3369) [reply .azure-pipelines/azure-pipelines-win.yml]: Ping...

**Comment by tk3369** [comment]
Hooray!  Thanks for the merge.
